src/Sylius/Component/Core/spec/Provider/ProductVariantsPricesProviderSpec.php

```diff
@@ -45,7 +46,7 @@ function it_provides_array_containing_product_variant_options_map_with_correspon
         ProductVariantInterface $blackSmallTShirt,
         ProductVariantInterface $whiteLargeTShirt,
         ProductVariantInterface $whiteSmallTShirt,
-        ProductVariantPriceCalculatorInterface $productVariantPriceCalculator
+        ProductVariantPricesCalculatorInterface $productVariantPriceCalculator
```

Comment:
We should also change it in the `let` method (it fails the build, as far as I see)